fuel/datasets/caltech101_silhouettes.py

```diff
@@ -39,4 +39,6 @@ def __init__(self, which_sets, size=28, load_in_memory=True, **kwargs):
 
     @property
     def data_path(self):
-        return os.path.join(config.data_path, self.filename)
+        for data_path in config.data_path:
+            if os.path.exists(os.path.join(data_path, self.filename)):
+                return os.path.join(data_path, self.filename)
```

Comment:
You should use `fuel.utils.find_in_data_path` instead of explicitly making the check here.